types/documentdb-server/index.d.ts

```diff
@@ -490,6 +490,11 @@ interface IRequest {
 	 */
     getBody(): Object;
     getBody<T>(): T;
+    /**
+     * Gets the OperationType for the request with a pre-trigger or post-trigger.
+     * Note: This will be one of 'Create', 'Replace', 'Upsert', or 'Delete'.
+     */
+    getOperationType(): string;
```

Comment:
if thats the case why not write the definition as 
```typescript
getOperationType(): "create" | "Replace" | "Upsert" | "Delete";
```

Reply:
Thank you @sheetalkamat! I've fixed this.